h2o-docs/src/product/data-science/algo-params/hglm.rst

```diff
@@ -64,7 +64,7 @@ Example
         # Set the response, predictor, and random columns
         y = "y"
         x = ["x1","x3","x5","x6"]
-        z = "Device"
+        z = 0
```

Comment:
nit: Maybe `z` could contain a list and directly assigned : `random_columns=z` as it's in R example.